test/integration/scheduler/queue_test.go

```diff
@@ -254,6 +254,40 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod2"),
 			enableSchedulingQueueHint: []bool{false, true},
 		},
+		{
```

Comment:
Can you add one more Pod which have nodeaffinity, but shouldn't be requeued in this test case? Then you can remove the second test case.

You can add pod2 that has NodeAffinity but not match even after node's update. And then make sure pod1 is requeued, while pod2 isn't.

Reply:
thanks for reviewing.
I changed the test code to combine them into one.